Formula/wireguard-tools.rb

```diff
@@ -27,8 +27,10 @@ class WireguardTools < Formula
   depends_on "wireguard-go"
 
   def install
-    inreplace ["src/completion/wg-quick.bash-completion", "src/wg-quick/darwin.bash"],
-              " /usr/local/etc/wireguard", " #{etc}/wireguard"
+    unless etc == "/usr/local/etc"
```

Comment:
`etc` is a `Pathname`, so you always have that `etc != "/usr/local/etc"`.

```suggestion
    if HOMEBREW_PREFIX.to_s != HOMEBREW_DEFAULT_PREFIX
```

Reply:
While that might work, I think it obscures the intent of the change.
```suggestion
    unless etc.to_s == "/usr/local/etc"
```